examples/editor/src/main.rs

```diff
@@ -26,11 +27,18 @@ fn app_view() -> impl View {
         .styling(SimpleStyling::new())
         .style(|s| s.size_full())
         .editor_style(default_dark_color)
-        .editor_style(move |s| s.hide_gutter(hide_gutter_a.get()));
+        .editor_style(move |s| {
+            s.hide_gutter(hide_gutter_a.get())
+                .render_white_space(RenderWhitespace::All)
+        });
     let editor_b = editor_a
         .shared_editor()
         .editor_style(default_dark_color)
-        .editor_style(move |s| s.hide_gutter(hide_gutter_b.get()))
+        .editor_style(move |s| {
+            s.hide_gutter(hide_gutter_b.get())
+                .render_white_space(RenderWhitespace::All)
+                .visible_whitespace(Color::BLACK)
```

Comment:
Why render it black? (or is this just to show off the extra styles?)

Reply:
Oh whoops. That was just testing. I'll remove that. 

I was doing it though because when there is an empty line I couldn't get any selection to show up. Any idea why that is or what would cause it?

Reply:
As in the texteditor was completely empty, and you couldn't get a selection to show up?  
Or just an empty line in it? (And by 'empty' do you mean 'with whitespace' or 'literally empty'
I'd assume a screenline problem, because I believe selection uses screen lines to decide whether to draw selection. Or just a problem in the logic for drawing selections.

Reply:
When there is an empty line (with a newline)
![image](https://github.com/lapce/floem/assets/13090441/516eecab-2f8e-4ef8-989f-0824df5ff2c3)
Everything is selected but the empty lines aren't showing

Reply:
Ah, that's a problem before your PR too.  
I think this is probably because I made visual lines not include the newline, and so they need to do an extra +1 on size for empty lines. I'll open a new issue.

Reply:
fixed